apps/antalmanac/src/components/Calendar/Toolbar/CustomEventDialog/ScheduleSelector.tsx

```diff
@@ -1,9 +1,8 @@
 import Checkbox from '@material-ui/core/Checkbox';
 import FormControlLabel from '@material-ui/core/FormControlLabel';
 import FormGroup from '@material-ui/core/FormGroup';
-import React, { PureComponent } from 'react';
-
 import type { RepeatingCustomEvent } from '@packages/antalmanac-types';
+import { ChangeEvent, PureComponent } from 'react';
```

Comment:
Did you intentionally import `ChangeEvent`? The `React` namescope should be available globally in React projects, and it's easier to use `React.ChangeEvent` for types.

```suggestion
import { type ChangeEvent, PureComponent } from 'react';
```